rcon/game_logs.py

```diff
@@ -4,42 +4,50 @@
 import sys
 import time
 import unicodedata
-from logging import config
+from typing import Callable, Dict, List
 
 from sqlalchemy import and_, desc, or_
-from sqlalchemy.exc import IntegrityError, InvalidRequestError
-from sqlalchemy.sql.expression import false, true
+from sqlalchemy.exc import IntegrityError
 
 from rcon.cache_utils import get_redis_client
-from rcon.commands import CommandFailedError, HLLServerError
 from rcon.config import get_config
 from rcon.discord import send_to_discord_audit
 from rcon.extended_commands import LOG_ACTIONS, Rcon
-from rcon.models import LogLine, PlayerName, PlayerSteamID, enter_session
-from rcon.player_history import (
-    add_player_to_blacklist,
-    get_player_profile,
-    player_has_flag,
-)
+from rcon.models import LogLine, PlayerSteamID, enter_session
+from rcon.player_history import (add_player_to_blacklist, get_player_profile,
+                                 player_has_flag)
 from rcon.recorded_commands import RecordedRcon
 from rcon.settings import SERVER_INFO
 from rcon.utils import FixedLenList
 
 logger = logging.getLogger(__name__)
 
-HOOKS = {
-    "TEAM KILL": [],
-    "CONNECTED": [],
-    "DISCONNECTED": [],
+HOOKS: Dict[str, List[Callable]] = {
+    "ADMIN BANNED": [],
+    "ADMIN KICKED": [],
+    "CAMERA": [],
+    "CHAT": [],
     "CHAT[Allies]": [],
+    "CHAT[Allies][Team]": [],
+    "CHAT[Allies][Unit]": [],
     "CHAT[Axis]": [],
-    "CHAT": [],
+    "CHAT[Axis][Team]": [],
+    "CHAT[Axis][Unit]": [],
+    "CONNECTED": [],
+    "DISCONNECTED": [],
     "KILL": [],
-    "CAMERA": [],
-    "TK": [],
-    "MATCH": [],
     "MATCH START": [],
     "MATCH ENDED": [],
+    "MATCH": [],
+    "TEAM KILL": [],
```

Comment:
You should move the teamkill / connect / disconnect back up. We want those to be executed quickly

Reply:
Whoops, I didn't even think about that, I sorted it so I had an easier time making sure I had each type in both files.